### PR TITLE
Update dataverse-broker.yaml

### PR DESCRIPTION
**Pull request by bruttozz**
add contents to register our davaverse-broker with the service catalog

### CODE REVIEW

File: openshift/dataverse-broker.yaml [hunk 1]
```diff
@@ -10,6 +10,14 @@ objects:
     name: dataverse-broker
   spec:
     url: http://dataverse-broker.bdc.svc.cluster.local
+    insecureSkipTLSVerify: false
+    caBundle: ${CA_BUNDLE}
```
Comment (pmorie) [inline openshift/dataverse-broker.yaml]: You can remove the `caBundle` and field for now - you don't need this until your broker serves https.
You should also remove `insecureSkipVerify`.

File: openshift/dataverse-broker.yaml [hunk 2]
```diff
@@ -10,6 +10,14 @@ objects:
...
+    authInfo: 
+      bearer: 
+        secretRef:
+          kind:      Secret
```
Comment (pmorie) [inline openshift/dataverse-broker.yaml]: You don't need the `kind` field here, but there's no harm in keeping it.

File: openshift/dataverse-broker.yaml [hunk 3]
```diff
@@ -10,6 +10,14 @@ objects:
...
+          name:      dataversebroker-client
```
Comment (pmorie) [inline openshift/dataverse-broker.yaml]: In order to have a static secret name with the service account token, you'll need to emulate this example: https://github.com/openshift/origin/blob/master/install/templateservicebroker/apiserver-template.yaml#L116

Comment (pmorie) [reply openshift/dataverse-broker.yaml]: You should refer to the service account token for `dataverse-broker` service account into the `dataversebroker-client` secret if you don't want to make other changes here.

**Comment by SamiSousa** [comment]
@bruttozz Should we close this pull request? Are all the changes here in #41 ?

**Pull request closed without merging**